app/src/main/java/com/example/flexioffice/data/BookingRepository.kt

```
@@ -31,6 +49,90 @@ class BookingRepository
+                // Hole Team-Informationen für den Reviewer
+                val team =
+                    try {
+                        val teamDoc =
+                            firestore
+                                .collection(Team.COLLECTION_NAME)
+                                .document(teamId)
+                                .get()
+                                .await()
+                        teamDoc.toObject(Team::class.java)
+                            ?: return Result.failure(IllegalArgumentException("Team nicht gefunden"))
+                    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)